### DISCUSSION TITLE
Should Z Probe offset flash on LCD when UBL active?

### DISCUSSION THREAD

**Discussion by andygodber** [mriscoc/Ender3V2S1#970] [Q&A] [answered]
Im sure I read somewhere that the Z Probe offset symbol (underlined "Z" with up arrow next to it) should flash when UBL is working.
I cant remember where I read it, but is it correct?

Ive never seen it flash so Im wondering if UBL is active!

**Comment by mriscoc** [comment, marked as answer]
At the end of this post there is a video: https://github.com/mriscoc/Ender3V2S1/wiki/Mesh-Leveling-Menu

**Reply by Crazy-Charles** [reply]
The Z probe blinking effect works well on the DWIN & DACAI screens but if you have a TJC screen it will not blink, this could be due to the icon background or the inability of the screen firmware to display transparency.

Filament runout blink also has the same issue, even on the DACAI screen as the particular stock ICON used is also lacking  - you can see the effect in the video link posted by mriscoc on the print speed icon, the colour behind it is "black"

Somewhere, I have a full collection of DWIN & DACAI stock icons that I spent hours reworking pixel by pixel brightening up the colours and making sure that the background colour was a pure black which is a requirement to make them "transparent"

**Comment by andygodber** [comment]
Thanks - does the icon flash all the time, while printing, or sometime else?
